src/Infrastructure/Translation/TranslatorInterface.php

```diff
@@ -6,6 +6,10 @@
 
 interface TranslatorInterface {
 
+	/**
+	 * @param string $messageKey
+	 * @param array<string, string|float|null> $parameters
```

Comment:
```suggestion
	 * @param array<string, scalar> $parameters
```
Although we'll be using this less and less (because more and more translation will move to the frontend), we still want to be as flexible as possible.